coins.json

```
@@ -1,1031 +1,1263 @@
 [
     {
-        "id": "nebulas",
-        "name": "Nebulas",
-        "symbol": "NAS",
-        "decimals": 18,
-        "blockchain": "Nebulas",
-        "derivationPath": "m/44'/2718'/0'/0/0",
-        "curve": "secp256k1",
-        "publicKeyType": "secp256k1Extended",
-        "explorer": "https://explorer.nebulas.io/#/tx/",
+        "id": "nebulas", 
+        "name": "Nebulas", 
+        "symbol": "NAS", 
+        "decimals": 18, 
+        "blockchain": "Nebulas", 
+        "derivationPath": "m/44'/2718'/0'/0/0", 
+        "curve": "secp256k1", 
+        "publicKeyType": "secp256k1Extended", 
+        "explorer": {
+            "url": "https://explorer.nebulas.io", 
+            "txPath": "/#/tx/",
+            "accountPath": "/#/address/"
+        }, 
         "info": {
-            "url": "https://nebulas.io",
-            "client": "https://github.com/nebulasio/go-nebulas",
-            "clientPublic": "https://mainnet.nebulas.io",
+            "url": "https://nebulas.io", 
```

A few trailing spaces
```suggestion
            "url": "https://nebulas.io",
```